src/mlx90632.c

```diff
@@ -475,14 +475,28 @@ int32_t mlx90632_init(void)
 int32_t mlx90632_addressed_reset(void)
 {
     int32_t ret;
+    uint16_t reg_ctrl;
+    uint16_t reg_value;
+
+    ret = mlx90632_i2c_read(MLX90632_REG_CTRL, &reg_value);
+    if (ret < 0)
+        return ret;
+
+    reg_ctrl = reg_value & ~MLX90632_CFG_PWR_MASK;
+    reg_ctrl |= MLX90632_PWR_STATUS_STEP;
+    ret = mlx90632_i2c_write(MLX90632_REG_CTRL, reg_ctrl);
```

Comment:
wouldn't it be better if we would just hardcode it to the power mode? That way we create constant environment for the command in case any other combination also has a problem...

```suggestion
    ret = mlx90632_i2c_write(MLX90632_REG_CTRL, MLX90632_PWR_STATUS_STEP);
```

Reply:
Since the issue is only in sleeping step mode, I would rather keep all the other settings in the control register and only change the mode to step. I do not really like fixing the value as it might lead to needlessly changing the settings back and forth.